readme.md

```diff
@@ -2,6 +2,11 @@
 
 > Check TypeScript type definitions
 
+## Overview
+
+This tool lets you write tests for your type definitions (i.e. your `.d.ts` files) by creating files with the `.test-d.ts` extension.
+
+These `.test-d.ts` files will not be executed, and not even compiled in the standard way. Instead, these files will be magically parsed for special constructs such as `expectError<Foo>(bar)` and then statically analyzed against your type definitions.
```

Comment:
```suggestion
These `.test-d.ts` files will not be executed, and not even compiled in the standard way. Instead, these files will be parsed for special constructs such as `expectError<Foo>(bar)` and then statically analyzed against your type definitions.
```

This is a good summary imo! Just one change to remove the word `magically` and it lgtm 😄

Reply:
@Ethan-Arrowood Thanks! Suggestion accepted!